packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -30,13 +30,8 @@ import 'localizations.dart';
 const double _kBackGestureWidth = 20.0;
 const double _kMinFlingVelocity = 1.0; // Screen widths per second.
 
-// An eyeballed value for the maximum time it takes for a page to animate forward
-// if the user releases a page mid swipe.
-const int _kMaxDroppedSwipePageForwardAnimationTime = 800; // Milliseconds.
-
-// The maximum time for a page to get reset to it's original position if the
-// user releases a page mid swipe.
-const int _kMaxPageBackAnimationTime = 300; // Milliseconds.
+// An value for the time it takes for a page to animate if the user releases a page mid swipe.
+const int _kDroppedSwipePageAnimationTime = 350; // Milliseconds.
```

Comment:
Small nit: Might make it easier to read if the entire duration was defined here instead. Then the `// Milliseconds.` comment wouldn't be needed.